crates/sc-consensus-subspace/src/notification.rs

```diff
@@ -29,7 +29,7 @@ type SharedNotificationSenders<T> = Arc<Mutex<Vec<TracingUnboundedSender<T>>>>;
 
 /// The sending half of the Subspace notification channel(s).
 #[derive(Clone)]
-pub(crate) struct SubspaceNotificationSender<T: Clone + Send + Sync + fmt::Debug + 'static> {
+pub struct SubspaceNotificationSender<T: Clone + Send + Sync + fmt::Debug + 'static> {
```

Comment:
Moving `MockBlockImport` into `sc-consensus-subspace` may not be enough since the slot notification also uses `SubspaceNotificationSender`. But I realized that the executor only required a stream produce item `(NumberFor<PBlock>, mpsc::Sender<()>)` and it doesn't have to be `SubspaceNotificationStream`, thus I use `TracingUnboundedReceiver` instead and remove the dependency of `sc-consensus-subspace`.